pkg/config/config_template.yaml

```diff
@@ -51,6 +51,13 @@ api_key:
 # Warning: percentiles must be specified as yaml strings
 #
 # histogram_percentiles: ["0.95"]
+#
+# Copy histgram values to distributions for true global distributions (in beta)
+# This will increase the number of custom metrics created
+# histogram_copy_to_distribution: true
+#
+# A prefix to add to distribution metrics created when histogram_copy_to_distributions is true
+# histogram_copy_to_distribution_prefix: ""
```

Comment:
what happens if a user enables the option while leaving the prefix empty, would the "duplicate" histogram vs distribution metrics be usable at all in DD? (if not we should set a more usable default value, but if they would still be perfectly usable then I'm fine with setting an empty string as the default value)

Reply:
Because historgram metrics are all suffixed, it is fine if they do not set a prefix.